fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -1225,10 +1224,11 @@ private boolean supportPartialPartitionQueryRewriteForExternalTable(Table table)
      *                    the update check.
      * @return          : Collect all need refreshed partitions of materialized view.
      */
-    public Set<String> getPartitionNamesToRefreshForMv(boolean isQueryRewrite) {
+    public boolean getPartitionNamesToRefreshForMv(Set<String> toRefreshPartitions,
```

Comment:
please update the method comment

Reply:
could we split this function into two function:
1. collectPartitionsToRefresh
2. canBeUsedForRewrite
?

Reply:
It's a good idea. However, it may increase the process time to check the materialized view's consistency.

we may shortcut the optimization time when MV Rewrite with union all is not set?